testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -13289,10 +13289,9 @@ private static void checkArrayConcatAggFuncFails(SqlOperatorFixture t) {
     f.check("select \"T\".\"X\"[2] "
             + "from (VALUES (ROW(ROW(3, CAST(NULL AS INTEGER)), ROW(4, 8)))) as T(x, y)",
         SqlTests.ANY_TYPE_CHECKER, isNullValue());
-    f.checkFails("select \"T\".\"X\"[1 + CAST(NULL AS INTEGER)] "
+    f.checkFails("select ^\"T\".\"X\"[1 + CAST(NULL AS INTEGER)]^ "
```

Comment:
Can we be more precise here, such as displaying `1+CAST (NULL AS INTEGER)` in the exception message?

Reply:
The position is included, that's what ^ stands for in these tests. The position is between the two carets.